website/templates/search_bar.mako

```diff
@@ -14,3 +15,31 @@
         </div>
     </div>
 </div>
+
+<script>
+/** Only for Search Bar Placeholder -- Allow IE and other browsers to work as the same */
+function placeholder(inputDom, inputLabel) {
+    inputDom.on('input', function () {
+        if (inputDom.val() === '') {
+            inputLabel.css( "visibility", "visible" );
+        } else {
+            inputLabel.css( "visibility", "hidden" );
+        }
+    });
+}
+$(document).ready(function() {
+    var inputDom =  $("#searchPageFullBar");
```

Comment:
Can you check for browser and only use this code if the browser is IE? I think this way we can avoid some side effects with other browser because there is some amount of intervention here in the dom.